docs/getting-started/src/flashing.md

```diff
@@ -22,8 +22,49 @@ STM32f411CEU6 microcontroller doesn't come with a programmer/debugger on board.
 <img src="assets/mcu-stlink-pins.png">
 </p>
 
-Connect the ST-Link and the MCU into your PC using USB cables and run the following command from command-line:
+Connect the ST-Link and the MCU into your PC using USB cables and run the following command from command-line inside cansat repository:
 
 ```
-cargo xtask embed cansat-stm32f4
+cargo xtask embed
 ```
+
+## Using standalone ST-LINK
+1. Connect the vires according to schematic below:
+<p>
+<img src="https://github.com/grupacosmo/cansat/assets/92270288/cedabacc-694a-4266-8d2e-87d94bb18612">
+</p>
+
+2. Connect the ST-Link and the MCU into your PC using USB cables and run the following command from command-line inside cansat repository:
```

Comment:
Connect the ST-Link and the MCU to your PC using USB cables. Then in the cansat directory run the following command from command-line